src/components/initiator.py

```diff
@@ -383,8 +383,13 @@ def update(self, minInterval, maxInterval, latency, timeout):
         self.updInitiatorRequest = self.__update(minInterval, maxInterval, latency, timeout) if not self.handles[0] == -1 else False;
         if self.updInitiatorRequest:
             if not self.peer is None:
+                # NOTE: Wait upto 3 intervals for Connection Parameter Response to be dispatched on air
+                # FIXME: Using 10 intervals to pass incorrect LL/CON/MAS/BV-27-C implementation, where TST is generating the LL_REJECT_EXT_IND
+                #        instead of controller detecting the Different Transaction Collision.
+                #        Zephyr controller as tester will not generate Connection Parameter Request if it is in Channel Map procedure
+                #        and waiting for instant to pass.
```

Comment:
Is this a limitation of using a regular controller (which cannot/should not exhibit illegal behavior) as tester, or is this caused by a bug in the Zephyr controller?